app/controllers/facility_user_reservations_controller.rb

```diff
@@ -21,9 +21,9 @@ def index
   # PUT /facilities/:facility_id/users/:user_id/reservations/:id/cancel
   def cancel
     order_detail = user_order_details.find(params[:id])
```

Comment:
It used to be `reservation_id` until b87258cd6a608b2c295133a99f4977740db8131a. Updating.